src/Relude/Extra/Foldable1.hs

```diff
@@ -103,6 +107,10 @@ class Foldable f => Foldable1 f where
     minimum1 :: Ord a => f a -> a
     minimum1 = SG.getMin #. foldMap1 SG.Min
 
+{- |
+
+@since 0.3.0
+-}
 instance Foldable1 NonEmpty where
```

Comment:
Adding `@since` annotations to instances makes the documentation even more beatiful ✨